src/app/workflow/dag/dag.component.html

```diff
@@ -32,6 +32,14 @@
             <input type="radio" name="dagType" [(ngModel)]="dagType" value="cwlviewer">Common Workflow Language Viewer
           </label>
         </div>
+        <div *ngIf="enableWdlViewer && (descriptorType$ | async) === ToolDescriptor.TypeEnum.WDL && (workflow$ | async)?.source_control_provider === 'GITHUB'" style="line-height: 20px;">
```

Comment:
1. Per my earlier comment, I don't think you need enableWdlViewer.
2. You have a `$isWDL` in workflow.query.ts -- is there a way you can just use that here (haven't looked at the code too closely yet)?
3. You shouldn't limit it to GitHub provider; you shouldn't care. It is like this for CWL viewer, because that external service only works with GitHub.